src/cryptography/utils.py

```diff
@@ -10,6 +10,8 @@
 import sys
 import warnings
 
+from pkg_resources import parse_version
```

Comment:
Doing this will trigger the import side effects we tried to avoid in #3347. Maybe we should do a simpler version check?